doc/nrf/ug_matter_tools.rst

```diff
@@ -26,6 +26,20 @@ Matter controller tools
 
 See the :ref:`ug_matter_configuring_controller` page for information about how to build and configure the Matter controller.
 
+ZAP tool
+********
+
+ZCL Advance Platform, in short ZAP tool, is a third-party tool that is a generic node.js-based templating engine for applications and libraries based on Zigbee Cluster Library.
+This tool is provided with the Matter submodule in the |NCS|.
+
+Use the ZAP tool to edit the ZAP file of your Matter application using GUI.
+A ZAP file is a JSON file that contains the data model configuration of clusters, commands, and attributes that are enabled for a given application.
+It is not used directly by the application, but it is used to generate global and customized configuration artifacts for handling the application configuration.
```

Comment:
`artifacts for handling the application configuration` sounds like helpers for processing the configuration itself which is not quite right. ZAP files are used to generate source code for processing requests enabled by the user (in the configuration). Maybe we could make it more concrete.

Reply:
Maybe we could add an example of what one can use this tool for in Matter? I mean, more specific examples than "configure clusters, commands, and attributes"?

Reply:
Examples are:
- enabling/disabling clusters, cluster commands, attributes and events
- configuring attributes' default values
- configuring attributes' properties, such as storage type (RAM storage, non-volatile storage, application-managed)

Reply:
Added to the section. Please check again, @Damian-Nordic .